pandas/tests/indexes/test_category.py

```diff
@@ -574,41 +574,56 @@ def test_reindexing(self):
             tm.assert_numpy_array_equal(expected, actual)
 
     def test_reindex_dtype(self):
-        c = CategoricalIndex(["a", "b", "c", "a"])
+        c = CategoricalIndex(["a", "b", "c"])
```

Comment:
Why these changes? Was the previous behavior incorrect?

I'd prefer to keep the diff in the tests as small as possible so that the changes here are easier to understand.

Reply:
You are right, I might actually have changed this unnecessarily. The changes to the `DataFrame.reindex` tests *are* necessary, but these changes might not be.